OpenRA.Mods.Common/Traits/World/EditorActorPreview.cs

```diff
@@ -79,8 +79,8 @@ public EditorActorPreview(WorldRenderer worldRenderer, string id, ActorReference
 
 			Footprint = ios?.OccupiedCells(Info, location, subCell) ?? new Dictionary<CPos, SubCell>() { { location, SubCell.FullCell } };
 
-			tooltip = Info.TraitInfos<EditorOnlyTooltipInfo>().FirstOrDefault(info => info.EnabledByDefault) as TooltipInfoBase
-				?? Info.TraitInfos<TooltipInfo>().FirstOrDefault(info => info.EnabledByDefault);
+			tooltip = Info.TraitInfos<EditorOnlyTooltipInfo>().FirstEnabledTraitOrDefault() as TooltipInfoBase
```

Comment:
I'm not sure if this works properly. We have Info classes here, but FirstEnabled*Trait*OrDefault only works on ConditionalTraits I think? (The function uses `IDisabledTrait` which is only implemented by `ConditionalTrait`.)

Reply:
![grafik](https://user-images.githubusercontent.com/7704140/125646768-4aa5bbef-75ce-4021-ac14-28d1c565c857.png)

Reply:
The debugger shows me that the Igloo TooltipInfo was always the first of the two, and it was always chosen as well.